src/components/FormContainer.tsx

```diff
@@ -15,17 +26,36 @@ const theme = createMuiTheme({
   }
 })
 
-const FormContainer = ({ onDone, onCancel, children }: PropsWithChildren<FormContainerProps>): ReactElement => (
+const FormContainer = ({
+  onDone,
+  onCancel,
+  children
+}: PropsWithChildren<FormContainerProps>): ReactElement => (
   <div>
     {children}
-    <Box display="flex" justifyContent="flex-start" paddingTop={2} paddingBottom={1}>
```

Comment:
I don't like this, attributes shouldn't be forced to a new line.

Reply:
It only does it if it exceeds 80 characters.